docs/src/development/file-transfer.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Transfer files to and from your app"
+weight: 7
+sidebarTitle: Transfer files
+---
+
+After your app is built, its file system is read-only.
+This means that the only way you can edit your app's files and code is through Git.
```

Comment:
:hammer_and_wrench: A lot of files aren't managed through Git, so I'd take that part out. Especially as it's somewhat contradicted by the next sentence.

```suggestion
This means that the only way you can edit your app's code is through Git.
```